generators/client/templates/vue/src/test/javascript/spec/app/admin/logs/logs.component.spec.ts.ejs

```diff
@@ -1,9 +1,9 @@
 import { shallowMount, createLocalVue, Wrapper } from '@vue/test-utils';
 import axios from 'axios';
-import Logs from '@/components/admin/logs/Logs.vue';
-import LogsClass from './Logs.component';
+import Logs from '../../../../../../main/webapp/app/components/admin/logs/logs.vue';
```

Comment:
It would be better to define what `@` means. In `tsconfig.json`, we should have something like:

```json
"paths": {
  "@/*": [
    "main/webapp/app/*"
  ]
},
```

Reply:
You are absolutely right, I'll fix that

Reply:
Great! Then it will be good for merging

Reply:
I can make this work when loading vue files but not for ts files, any advises ?

Reply:
Obviously ! Thanks, I'll fix this part asap !